### ISSUE TITLE
Moving content updates modifiedTime and modifier properties in content only after first move

### ISSUE THREAD

**Issue by alansemenov** [enonic/xp#5874]
1. Why do we even have to update modifiedTime and modifier when we move content? Shouldn't this only use node.move operation without modifiying content data?
2. Why are these fields updated only on first move? Any consecutive move operation won't update the fields.

We are considering to implement a new Move permission that will allow user to move content. Currently we check for Delete to allow moving content and for Create in target location.

https://discuss.enonic.com/t/content-permissions/951

First step is to answer to the ticket. Then we will create a new issue for implementation.

**Comment by runarmyklebust** [comment]
When I have tested this, I'm able to reproduce this when moving out of site and then changing the xtra-data of the content -> If this happens you will get an updated modified-time since the content is actually modified when moving. 

Does this make sense with your observed behaviour?

**Comment by runarmyklebust** [comment]
Works as expected, modified is updated if x-data is changed (e.g moved out of site)
